apps/api/src/index.ts

```diff
@@ -71,54 +137,14 @@ app.get("/", (_req, res) => {
 // Apply error handler
 app.use(errorHandler);
 
-// Start server
-const startServer = async () => {
```

Comment:
No need to wrap all the code in a function, you can just run it all top level, awaits and everything.